CAFAna/Core/Hist.cxx

```diff
@@ -154,6 +154,16 @@ namespace ana
     return ret;
   }
 
+  //----------------------------------------------------------------------
+  Hist Hist::AdoptErrors(Eigen::ArrayXd&& v, Eigen::ArrayXd&& e)
```

Comment:
Can you include the `.h` change in this same PR so it can all be merged together?